### ISSUE TITLE
[Request] Option to generate with bitcode enabled

### ISSUE THREAD

**Issue by phlippieb** [schickling/Icomoon.swift#2]
iTunes connect has recently been rejecting app submissions if some of the app's frameworks have bitcode enabled, and others do not. So if someone who uses Icomoon should want to enable bitcode for their project, it's important that they can also generate their Icomoon framework with bitcode enabled. It would be awesome if this could simply be specified as a parameter to the Icomoon command.

**Comment by schickling** [comment]
I'm happy to merge this feature in. Would you mind providing a PR?

**Comment by nitrag** [comment]
This should be resolved. Please close.

**Comment by phlippieb** [comment]
@nitrag Awesome! Saw that you merged a PR for this. Is bitcode _always_ enabled now, though?

**Comment by nitrag** [comment]
So I've been using this in one of my apps and I had made several commits a
while back to ensure bitcode is enabled. It was a struggle. Now that I'm
going through and using it for another app I'm beefing up the docs. It
*should* be enabled now but wont know for sure if there's any other steps
to ensure bitcode is enabled until I submit my new app.

TLDR: Yes I'm pretty sure.
